pyxs/helpers.py

```diff
@@ -125,6 +126,16 @@ def validate_path(path):
     return path
 
 
+def path_exists(path):
+    """Checks if a given path exists, i.e. is an actual Xenstore path.
+    :exc:`~pyxs.exceptions.InvalidPath` for details
+    """
+    try:
+        return Client().ls(path) != None
```

Comment:
Are you sure you want it as a function and not as a client method, i.e. `Client.exists`. Also, please use `is not` for comparison with `None`.